stable/zeppelin/values.yaml

```diff
@@ -14,3 +14,16 @@ spark:
   driverMemory: 1g
   executorMemory: 1g
   numExecutors: 2
+
+ingress:
+  enabled: false
+  certManager: false
+  host: zeppelin.local
+  path: /
+  tls: false
```

Comment:
Just another comment otherwise lgtm

can you make this ingress section similar to the other charts?

you dont need to define the certmager, this will be part of the annotations, same for the basic auth

for example this was from the elasticsearch.
```
  ingress:
    enabled: false
    annotations: {}
      # kubernetes.io/ingress.class: nginx
      # kubernetes.io/tls-acme: "true"
    path: /
    hosts:
      - chart-example.local
    tls: []
    #  - secretName: chart-example-tls
    #    hosts:
    #      - chart-example.local
```

Reply:
yeah, another will be fine. thanks for this!